addons/stock/models/stock_picking.py

```diff
@@ -343,14 +386,41 @@ def get_action_picking_tree_ready(self):
     def get_action_picking_type_operations(self):
         return self._get_action('stock.action_get_picking_type_operations')
 
+    def get_action_picking_type_moves_analysis(self):
+        action = self.env["ir.actions.actions"]._for_xml_id('stock.stock_move_action')
+        action['domain'] = (action['domain'] or []) + [('picking_type_id', '=', self.id)]
+        return action
+
     def get_stock_picking_action_picking_type(self):
         return self._get_action('stock.stock_picking_action_picking_type')
 
+    def get_action_picking_type_ready_moves(self):
+        return self._get_action('stock.action_get_picking_type_ready_moves')
+
     @api.depends('code')
     def _compute_show_picking_type(self):
         for record in self:
             record.show_picking_type = record.code in ['incoming', 'outgoing', 'internal']
 
+    def _compute_description(self):
+        for picking_type in self:
+            if picking_type.description:
+                return
+            picking_type.description = ''
+            match picking_type.code:
+                case 'incoming':
+                    picking_type.description = _(
+                        'Receive products from suppliers. Use replenishment rules to avoid stockouts.'
+                    )
+                case 'outgoing':
+                    picking_type.description = _(
+                        'Send products to your customers and ship them with an integrated carrier.'
+                    )
+                case 'internal':
+                    picking_type.description = _(
+                        'Move products within a warehouse. Use the barcode app for more convenience.'
```

Comment:
barcode app is enterprise. It's a kind of fuck to community. I would override it in enterprise

Reply:
This is no longer an issue: description field was removed.